README.md

```diff
@@ -1,2 +1,71 @@
-# conference-page
-HTML/CSS &amp; JavaScript capstone project - Conference page
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Portfolio: setup and mobile version skeleton.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML, CSS
+- Stylelint, Web Hint, Lighthouse
+
+## Live Demo
+
+[Live Demo Link](https://topeogunleye.github.io/conference-page/)
```

Comment:
- [ ] As the requirement says you should have recorded a loom video telling about the project.

![vvvvvv](https://user-images.githubusercontent.com/61562626/142161094-2535bd4d-33ce-4107-8bdb-b9906a75003a.jpeg)